libs/render_delegate/utils.cpp

```diff
@@ -681,30 +681,24 @@ void HdArnoldGetPrimvars(
             // The number of motion keys has to be matched between points and normals, so if there are multiple
             // position keys, so we are forcing the user to use the SamplePrimvars function.
             if (multiplePositionKeys && primvarDesc.name == HdTokens->normals) {
+                HdArnoldInsertPrimvar(primvars, primvarDesc.name, primvarDesc.role, primvarDesc.interpolation, 
 #ifdef USD_HAS_SAMPLE_INDEXED_PRIMVAR
```

Comment:
May be we could remove this ifdef as it was used for very old version ?

Reply:
it's needed for versions older than 21.05, so I think we still need it ?